src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -82,6 +82,6 @@ var ReactDOMUnknownPropertyDevtool = {
   onDeleteValueForProperty(node, name) {
     warnUnknownProperty(name);
   },
-}
+};
```

Comment:
Yeah it failed locally for me so I had to fix it. But somehow it didn't fail in the PR that added this file.